CONTRIBUTING.md

```diff
@@ -1,68 +1,33 @@
 # Contributing Guidelines
 
-Thanks for your interest in contributing to `ogr`.
-
-The following is a set of guidelines for contributing to `ogr`.
-Use your best judgement, and feel free to propose changes to this document in a pull request.
-
-By contributing to this project you agree to the Developer Certificate of Origin (DCO). This document is a simple statement that you, as a contributor, have the legal right to submit the contribution. See the [DCO](DCO) file for details.
+Please follow common guidelines for our projects [here](https://github.com/packit/contributing).
 
 ## Reporting Bugs
 
-Before creating bug reports, please check a [list of known issues](https://github.com/packit/ogr/issues) to see
-if the problem has already been reported (or fixed in a master branch).
-
-If you're unable to find an open issue addressing the problem, [open a new one](https://github.com/packit/ogr/issues/new).
-Be sure to include a **descriptive title and a clear description**. Ideally, please provide:
-
-- version of ogr you are using (`rpm -q python3-ogr` or `pip3 freeze | grep ogr`)
-- the command you executed and a debug output (using option `--debug`)
-
-If possible, add a **code sample** or an **executable test case** demonstrating the expected behavior that is not occurring.
-
-**Note:** If you find a **Closed** issue that seems like it is the same thing that you're experiencing, open a new issue and include a link to the original issue in the body of your new one.
-You can also comment on the closed issue to indicate that upstream should provide a new release with a fix.
-
-### Suggesting Enhancements
-
-Enhancement suggestions are tracked as GitHub issues.
-When you are creating an enhancement issue, **use a clear and descriptive title** and **provide a clear description of the suggested enhancement** in as many details as possible.
+- [List of known issues](https://github.com/packit/ogr/issues)
+- Getting a version of `ogr`:<br>
+  `rpm -q python3-ogr` or `pip3 freeze | grep ogr`
+- [In case you need to create a new issue](https://github.com/packit/ogr/issues/new)
 
-## Guidelines for Developers
-
-If you would like to contribute code to the `ogr` project, this section is for you!
-
-### Is this your first contribution?
-
-Please take a few minutes to read GitHub's guide on [How to Contribute to Open Source](https://opensource.guide/how-to-contribute/).
-It's a quick read, and it's a great way to introduce yourself to how things work behind the scenes in open-source projects.
-
-### Dependencies
-
-If you are introducing a new dependency, please make sure it's added to:
-
-- [setup.cfg](setup.cfg)
-
-### Documentation
+## Documentation
 
 If you want to update documentation, [README.md](README.md) is the file you're looking for.
 
-#### Changelog
+## Links to documentation
```

Comment:
```suggestion
## Documentation of the service APIs
```

... or something else to make it clear that we are speaking about documentation of Github/GitLab/Pagure.